app/script/localization/strings.coffee

```diff
@@ -302,7 +302,7 @@ z.string.invite_meta_key_pc = 'Ctrl'
 z.string.invite_hint_selected = 'Press %meta_key + C to copy'
 z.string.invite_hint_unselected = 'Select and Press %meta_key + C'
 z.string.invite_headline = 'Invite people to Wire'
-z.string.invite_message = 'I’m on Wire, search for %username or visit wire.com/download.'
+z.string.invite_message = 'I’m on Wire, search for %username or visit get.wire.com.'
```

Comment:
@gregor I'll push an extra commit to remove the protocol from the remaining instance.